sqlvm-alwayson-cluster/azuredeploy.json

```diff
@@ -0,0 +1,1084 @@
+{  
```

Comment:
Readme should be filled out. Our sample readme is a good place to start https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/sample-README.md